awx/main/utils/licensing.py

```diff
@@ -383,9 +384,18 @@ def validate(self):
             current_instances = Host.objects.active_count()
         else:
             current_instances = 0
+        if HostMetric:
+            automated_instances = HostMetric.objects.count()
+            first_host = HostMetric.objects.order_by('first_automation').first()
+            automated_since = first_host.first_automation
+        else:
+            automated_instances = 0
+            automated_since = now()
         instance_count = int(attrs.get('instance_count', 0))
         attrs['current_instances'] = current_instances
-        free_instances = instance_count - current_instances
+        attrs['automated_instances'] = automated_instances
+        attrs['automated_since'] = automated_since
+        free_instances = instance_count - automated_instances
```

Comment:
This changes the definition of `free_instances` to be based on the HostMetric automated instances.